app/models/ext_management_system.rb

```diff
@@ -36,6 +36,10 @@ def self.supported_types_and_descriptions_hash
     supported_subclasses.each_with_object({}) { |klass, hash| hash[klass.ems_type] = klass.description }
   end
 
+  def self.subclasses_support?(feature)
+    concrete_subclasses.select { |subclass| subclass.supports?(feature) }
```

Comment:
Why `concrete_subclasses` here as opposed to `supported_subclasses`?  Not sure it makes sense for us to lookup providers they can't possible add.